### PR TITLE
Tickets/pipe2 d 89

### PR DESCRIPTION
**Pull request by ziggyman**

### CODE REVIEW

File: src/FiberTraces.cc [hunk 1]
```diff
@@ -319,7 +396,7 @@ namespace pfsDRPStella = pfs::drp::stella;
       *varIt = sum(_trace->getVariance()->getArray()[i]);
       *maskIt = sum(_trace->getMask()->getArray()[i]);
     }
-    if ( !spectrum->setSpectrum( spec ) ){
+    if (!spectrum->setSpectrum(spec)){
```
Comment (RobertLuptonTheGood) [inline src/FiberTraces.cc]: I think that this error condition is impossible.  If so, the check should be removed or replaced by an assertion.

File: src/FiberTraces.cc [hunk 2]
```diff
@@ -388,14 +472,17 @@ namespace pfsDRPStella = pfs::drp::stella;
       throw LSST_EXCEPT(pexExcept::Exception, message.c_str());
     }
     ///TODO: change to sqrt(T_A2_VarianceArray)
```
Comment (RobertLuptonTheGood) [inline src/FiberTraces.cc]: Please add JIRA ticket number for this todo to comment

File: src/FiberTraces.cc [hunk 3]
```diff
@@ -106,85 +119,135 @@ namespace pfsDRPStella = pfs::drp::stella;
...
+      cout << "FiberTrace" << _iTrace << "::setFiberTraceProfileFittingControl: ";
+      cout << "fiberTraceProfileFittingControl->xCorProf = <";
+      cout << fiberTraceProfileFittingControl->xCorProf << ">" << endl;
     #endif
 
     if (!fiberTraceProfileFittingControl->isClassInvariant()){
-      string message("FiberTrace::setFiberTraceProfileFittingControl: ERROR: fiberTraceProfileFittingControl is not ClassInvariant");
+      string message("FiberTrace::setFiberTraceProfileFittingControl: ");
+      message += "ERROR: fiberTraceProfileFittingControl is not ClassInvariant";
       cout << message << endl;
```
Comment (ziggyman) [inline src/FiberTraces.cc]: See PIPE2D-170

File: src/FiberTraces.cc [hunk 4]
```diff
@@ -1,9 +1,9 @@
 #include "pfs/drp/stella/FiberTraces.h"
 
-namespace pfsDRPStella = pfs::drp::stella;
+namespace stella = pfs::drp::stella;
```
Comment (ziggyman) [inline src/FiberTraces.cc]: Filed ticket PIPE2D-194

File: src/FiberTraces.cc (hunk 1 above)
Comment (ziggyman) [inline src/FiberTraces.cc]: filed ticket PIPE2D-195

File: src/FiberTraces.cc (hunk 2 above)
Comment (ziggyman) [inline src/FiberTraces.cc]: filed ticket PIPE2D-196

File: include/pfs/drp/stella/FiberTraces.h [hunk 5]
```diff
@@ -55,7 +55,7 @@
 //#define __DEBUG_TRACEFUNC__
 //#define __DEBUG_UNIQ__
 //#define __DEBUG_XCENTERS__
-#define DEBUGDIR "/Users/azuri/spectra/pfs/2014-11-02/debug/"// /home/azuri/entwicklung/idl/REDUCE/16_03_2013/"//stella/ses-pipeline/c/msimulateskysubtraction/data/"//spectra/elaina/eso_archive/red_564/red_r/"
+#define DEBUGDIR "/Users/azuri/spectra/pfs/2014-11-02/debug/"
```
Comment (ziggyman) [inline include/pfs/drp/stella/FiberTraces.h]: Removed as it was obsolete

**Pull request closed without merging**